ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/management-security.j2

```diff
@@ -129,6 +129,30 @@
 | {{ policy.name }} | > {{ min_digits }} | > {{ min_length }} | > {{ min_lower }} | > {{ min_special }} | > {{ min_upper }} | < {{ max_repetitive }} | < {{ max_sequential }} |
 {%         endfor %}
 {%     endif %}
+{%     if management_security.shared_secret_profiles is arista.avd.defined %}
+
+### Secret Profiles
+{%         for profile in management_security.shared_secret_profiles | arista.avd.natural_sort("profile") %}
+
+#### {{ profile.profile }}
+
+| Secret Name | Secret Type | Receive Lifetime | Transmit Lifetime |
```

Comment:
The Secret type / "7" in the table has no real value. Please remove that column. Thanks.

Reply:
Removed secrete_type key from table.